src/app/courses/components/add-course/add-course.component.html

```diff
@@ -0,0 +1,82 @@
+<section class="add-course-container">
+    <header class="add-course-header">
+        <h2 class="add-course-title">add course</h2>
+        <div class="separator"></div>
+        <!-- hidden for now -->
+        <!-- <div class="close-btn" role=button>
+            <fa-icon icon="times"></fa-icon>
+        </div> -->
+    </header>
+    <form class="add-course-form">
+        <ul class="form-sections">
+            <li class="course-title-container">
+                <label for="course-title-id">title</label>
+                <input
+                    type="text"
+                    id="course-title-id"
+                    class="course-title"
+                    placeholder="Enter a course name"
+                    [(ngModel)]="courseTitle"
+                    name="title">
+            </li>
+            <li class="course-description-container">
+                <app-input-text></app-input-text>
```

Comment:
It should be just textarea, placed here

Reply:
Is it a strict requirement that `textarea` shouldn't be presented as a standalone component?

Reply:
It just a note, that in task was highlighted, which components should be separate. And textarea was not in that list.

Reply:
I see, thanks.